static/css/drawer.css

```diff
@@ -164,11 +165,21 @@ h1, legend,
     margin-top: 0;
     margin-left: 42px;
     padding-top: 10px;
-    padding-right: 150px;
+    padding-right: 90px;
     white-space: nowrap;
     text-overflow: ellipsis;
     overflow: hidden;
 }
+#iob-cob h1 {
```

Comment:
these changes must be from some rebase. i need to add only color to legend. legend was not displayed properly on IE.
How to solve it?